src/components/@shared/atoms/Modal/index.tsx

```diff
@@ -1,8 +1,6 @@
 import React, { ReactElement, ReactNode } from 'react'
 import ReactModal from 'react-modal'
-import styles from './Modal.module.css'
-
-if (process.env.NODE_ENV !== 'test') ReactModal.setAppElement('#__next')
```

Comment:
FYI, the solution here was not to remove this line but to prevent it from firing when run in storybook, as that would make everything fail as there is no `#__next` id. Storybook does not set `NODE_ENV=test` like a Jest run does automatically, so I added this manually to the npm command